bugbug/phabricator.py

```diff
@@ -291,3 +292,17 @@ def get_pending_review_time(rev: RevisionDict) -> timedelta | None:
         return datetime.utcnow() - last_exclusion_end_date
     else:
         return datetime.utcnow() - creation_date
+
+
+def fetch_diff(revision_id, patch_id):
+    try:
+        url = f"https://phabricator.services.mozilla.com/D{revision_id}?id={patch_id}&download=true"
+        response = requests.get(url)
+        response.raise_for_status()
+        return response.text
+    except requests.HTTPError as e:
+        logger.error(f"HTTP error fetching diff: {e}")
+        return None
+    except Exception as e:
+        logger.error(f"Unexpected error: {e}")
+        return None
```

Comment:
We already have a function to do this. We may want to consider reusing it: https://github.com/mozilla/bugbug/blob/6134ff622c1ee46f323439f77a08a18726912741/scripts/inline_comments_data_collection.py#L69-L75

Reply:
Done here: a4c1b5deb73e2ea5b1753118034bc764c88804c1